vyper/compiler/output.py

```diff
@@ -134,6 +134,13 @@ def build_interface_output(compiler_data: CompilerData) -> str:
             encoded_args = "\n    ".join(f"{name}: {typ}" for name, typ in event.arguments.items())
             out = f"{out}event {event.name}:\n    {encoded_args if event.arguments else 'pass'}\n"
 
+    if interface.structs:
```

Comment:
prefer `if len(interface.structs) > 0:`

i see that's how it's done in the rest of the function, i'll push a fix

Reply:
b2a64a2da84de66442a20400ce7701c1c4ac792d